firefox-ios/Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/CreditCardBottomSheetHeaderView.swift

```diff
@@ -55,10 +51,7 @@ class CreditCardBottomSheetHeaderView: UITableViewHeaderFooterView, ReusableCell
         label.lineBreakMode = .byWordWrapping
         label.setContentHuggingPriority(.defaultHigh, for: .vertical)
         label.text = String(format: String.CreditCard.RememberCreditCard.Header, AppName.shortName.rawValue)
-
-        label.font = DefaultDynamicFontHelper.preferredFont(
-            withTextStyle: .body,
-            size: UX.headerLabelFontSize)
+        label.font = FXFontStyles.Regular.subheadline.scaledFont()
```

Comment:
@cwzilla can you confirm that this change is okay for the string `Firefox encrypts your card number. Your security code won’t be saved.`? It used to be size 15 with text style `body`, but is now using `subheadline` which is also size `15`.

Reply:
That looks good!

Reply:
awesome, thanks for verifying!